test/inductor/test_compiled_optimizers.py

```diff
@@ -163,7 +163,7 @@ def tearDown(self):
     test_adamw = make_test(AdamW, lr=0.01)
     # Need to an impl which does not use python scalars
     # test_adamax = make_test(Adamax, lr=0.01)
-    # test_nadam = make_test(NAdam, lr=0.01)
+    test_nadam = make_test(NAdam, lr=0.01)
```

Comment:
can you add more configs as in https://github.com/pytorch/pytorch/blob/main/test/optim/test_optim.py#L864-L879

Reply:
I don't think that's for this PR, I was merely adding something that I already knew we supported (nadam)

I can add additional tests for those too, I'm not sure if lr as a tensor works atm, I'll have to try it with all of them.